examples/pylab_examples/arrow_demo.py

```diff
@@ -165,10 +163,9 @@ def draw_arrow(pair, alpha=alpha, ec=ec, labelcolor=labelcolor):
         else:
             length = max_arrow_length
         # set the transparency of the arrow
-        if display == 'alph':
+        if display == 'alpha':
```

Comment:
As was this mistake, too. In fact, this code, I think, became the basis for our FancyArrow class. We should see if this mistake exists in that code, too.